src/core/entities/MRTextAreaEntity.js

```diff
@@ -78,8 +78,8 @@ export class MRTextAreaEntity extends MRTextInputEntity {
     /**
      * Overrides the connected method to include setup for handling multiline text.
```

Comment:
this function needs proper jsdoc descriptions not just function body

@function
@description

and - can you add `(async)` to the description like you did for the other newly async functions?